code/modules/unit_tests/create_and_destroy.dm

```diff
@@ -21,10 +21,17 @@
 		/obj/effect/mapping_helpers/component_injector,
 		//Singleton
 		/mob/dview,
+		//Xenobio basetypes
+		/obj/item/slimecross/gentle,
+		//just no
+		/atom/movable/plane_master_controller
```

Comment:
So it isn't just an obscure comment, what's the actual reasoning for this?

Reply:
oops forgot to replace the comment

Reply:
the reason for it to be blacklisted is simply cause this can not actually exist on its own at all i was thinking leaving it in there simply alone but ok maybe i should leave it in for the xenobio basetype simply cause this is not a real object is the baseline and ironcally the only one of them runtiming

Reply:
ok i originally did so because of what gets forwarded but i will just handle this on the new proc i guess